app/src/main/java/com/github/sdp/mediato/MainActivity.java

```diff
@@ -96,15 +96,11 @@ private void setDefaultFragment(String username) {
     searchFragment = new SearchFragment();
     exploreFragment = new ExploreFragment();
 
-    Review review = new Gson().fromJson(
-            getIntent().getStringExtra("review"), Review.class);
```

Comment:
Why is the review encoded as a json string ?
Would it not be better to be in a bundle : [https://stackoverflow.com/a/14333555](https://stackoverflow.com/a/14333555)

or  a parcelable [https://stackoverflow.com/a/2141166](https://stackoverflow.com/a/2141166)

Reply:
Yeah that's why I removed it this code is no longer needed and is put in a bundle elsewhere.